sktime/utils/_testing/scenarios.py

```diff
@@ -0,0 +1,200 @@
+# -*- coding: utf-8 -*-
+"""Testing utility to play back usage scenarios for estimators.
+
+Contains TestScenario class which applies method/args subsequently
+"""
+
+__author__ = ["fkiraly"]
+
+__all__ = ["TestScenario"]
+
+
+from copy import deepcopy
+from inspect import isclass
+
+import numpy as np
+
+
+class TestScenario:
+    """Class to run pre-defined method execution scenarios for objects.
+
+    Parameters
+    ----------
+    args : dict of dict, default = None
+        dict of argument dicts to be used in methods
+        names for keys need not equal names of methods these are used in
+            but scripted method will look at key with same name as default
+        must be passed to construtor, or set in a child class
+    default_method_sequence : list of str, default = None
+        optional, if given, default method sequence to use in "run"
+        if not provided, "run" must alwayss be passed a method_sequence
```

Comment:
1. fixing minor typos.
2. in current way `run` is written, `method_sequence` does not *need* to be provided, as it can copy arg_sequence. either update docstring here or (my preference) simplify the logic a little in `run` by having copying only be one way. Explicitly, in `run`, if `arg_sequence` is not provided, then `arg_sequence` copies `method_sequence`, but not vice versa

```suggestion
    default_method_sequence : list of str, default = None
        optional, if given, default method sequence to use in `run`
        if not provided, `run` must always be passed a method_sequence
```

Reply:
fixed, clarified requirement/defaulting behaviour